pkg/api/latest/latest.go

```diff
@@ -30,8 +31,21 @@ import (
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/util"
 )
 
+func init() {
+	version := os.Getenv("KUBE_API_LATEST_VERSION")
```

Comment:
Is an env var better than a command-line flag? The benefit of the flag is that the default value is clear-- it's documented for us already.